pkg/ebpf/netlink/conntracker.go

```diff
@@ -159,13 +161,13 @@ func newConntrackerOnce(procRoot string, deleteBufferSize, maxStateSize int) (Co
 	return ctr, nil
 }
 
-func (ctr *realConntracker) GetTranslationForConn(ip util.Address, port uint16) *IPTranslation {
+func (ctr *realConntracker) GetTranslationForConn(ip util.Address, port uint16, proto uint8) *IPTranslation {
```

Comment:
Any reason we can't just use `ConnectionType` here instead of uint8?

Reply:
I had originally tried to use `ebpf.ConnectionType` but it caused a cyclic dependency between `ebpf` and `ebpf/netlink`.  


We could use the `process.ConnectionType` but I would still need to convert between `connection` and `process` types.  Now that I'm tying it out, I think that's a little easier to follow. I'll do that.  


Long term, I think we should try getting rid of `ebpf.ConnectionType` together in favor of `process.ConnectionType` but that's  a bigger code change.